src/include/catalog/column.h

```diff
@@ -77,7 +77,7 @@ class Column {
   auto IsInlined() const -> bool { return column_type_ != TypeId::VARCHAR; }
 
   /** @return a string representation of this column */
-  auto ToString() const -> std::string;
+  auto ToString(bool simplified = true) const -> std::string;
```

Comment:
I think Andy generally does not prefer having default arguments and historically they have been a bit of an issue, but I don't feel very strongly about it in this specific use-case. Just a FYI

Reply:
I think we can use the simplified representation as the default one, and move the previous ToString to `ToStringVerbose`.

Reply:
I'll change this later in batch, revisiting all such occurrences and see if we can simplify `ToString` of other classes.